### ISSUE TITLE
DPL-920 Transfer from 10X chip to GEMs plate

### ISSUE THREAD

**Issue by KatyTaylor** [sanger/limber#1393]
**User story**
As a user of the scRNA Core pipeline, I would like LIMS support for the transfer from the 10X chip to the GEMs plate.

**Who are the primary contacts for this story**
Katy, Lesley, Abby

**Who is the nominated tester for UAT**
Lesley, Abby

**Acceptance criteria**
To be considered successful the solution must allow:
- [x] Input chip is "LRC GEM-X 5p Chip"
- [x] Output plate is "LRC GEM-X 5p GEMs"
- [x] It is a straight stamp (although the row on the chip is transposed to a column on the plate)
- [x] Bed verification:
    - [x] Input chip on bed 15
    - [x] Output plate on bed 5
- [x] Integration Suite test is updated to include this transfer.

**Related issues**
Probably best to NOT do this in parallel with https://github.com/sanger/limber/issues/1392, as some work to do with implementing the new chip might be duplicated.

**Comment by KatyTaylor** [comment]
Updated in anticipation of the GEM-X decision. (1:1 transfer rather than 1:2)

**Comment by KatyTaylor** [comment]
Removed on hold label following decision to go with GEM-X chip.

**Comment by andrewsparkes** [comment]
_It is a straight stamp (although the row on the chip is transposed to a column on the plate)_

That sounds like we need a labware creator that consolidates to top left rather than a stamp. Check if we have one that just does that.
Q. what if we have gaps on the chip plate row, can we consolidate?

**Comment by yoldas** [comment]
LRC GEM-X 5p Chip will look like a single row on the presenter. Do we need to transpose it back to a column on LRC GEM-X 5p GEMs? Is it a convention?

LRC PBMC Pools is compressed to top-left (side effect of creating pools). When the column is transposed to LRC GEM-X 5p Chip, it does not leave gaps. Transferring that to GEM-X 5p GEMs should also have no gaps.

LRC PBMC Pools Input may have gaps. We might need to clarify that.

**Comment by andrewsparkes** [comment]
Labware creator `app/models/labware_creators/partial_stamped_plate_without_dilution.rb` is almost what we need I think. This labware creator does the compression. It also uses well filter to take a partial subset of wells from the parent, which we don't need in this story.

Assuming compression is ok, I suggest refactoring this.
Currently: 
**partial_stamped_plate_without_dilution** extends **stamped_plate**

Proposed:
**partial_stamped_plate_without_dilution** extends **stamped_plate_compressed** (which we use in this story) which extends **stamped_plate**
(and maybe rename **partial_stamped_plate_without_dilution** to **stamped_plate_compressed_partial**)

**Comment by andrewsparkes** [comment]
@KatyTaylor Do we know which type of robot for this bed verification? Hamilton, Bravo etc?
